spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1692,13 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+When returning the set of directives from the `__Schema` introspection type, both
```

Comment:
Thanks for the fixes.
@sungam3r  Looking more into the current state of the spec I think we need to merge this text with this part:
https://graphql.github.io/graphql-spec/draft/#sel-DAHnBDPAADRC9Bnwe

Because we duplicate a lot of stuff and have totally unrelated things in the middle.
Similar to #597 I would suggest adding `Built-in Directives` section:
https://github.com/graphql/graphql-spec/pull/597/files#diff-86a7cb172ae1d944ca3719bd848b1d8bR347

Reply:
Plus it will allow us to write `built-in directives` instead of explicitly listed them everytime.

Reply:
done